backend/service/chaos/experimentation/experimentstore/experiment_store.go

```diff
@@ -24,7 +24,7 @@ import (
 const Name = "clutch.service.chaos.experimentation.store"
 
 // ExperimentStore stores experiment data
-type ExperimentStore interface {
+type ExperimentStorer interface {
```

Comment:
While we're at it, might as well also avoid stutter/repetition per convention.

See [Go Blog: Package Names -- Naming package contents](https://blog.golang.org/package-names#TOC_3.)

```suggestion
type Storer interface {
```

Reply:
@danielhochman could you take another look at this one, please?